app/lib/features/bug_report/pages/bug_report_page.dart

```diff
@@ -56,10 +56,10 @@ class _BugReportState extends ConsumerState<BugReportPage> {
       final Map<String, String> extraFields = {};
       if (submitErrorAndStackTrace) {
         if (widget.error != null) {
-          extraFields['error'] = widget.error.toString();
+          extraFields['error'] = widget.error!;
         }
         if (widget.stack != null) {
-          extraFields['stack'] = widget.stack.toString();
+          extraFields['stack'] = widget.stack!;
         }
```

Comment:
So, while removing a lot of `!` in other Para you are adding them again here? I believe this is just to make the linter happy!?!

You know if you do 
```
final error = widget.error;
if (error != null) {
  extra fields['error'] = error;
}
```